mailpoet/lib/Config/Populator.php

```diff
@@ -530,17 +530,17 @@ protected function populate($model) {
     }
   }
 
-  private function rowExists($table, $columns) {
+  private function rowExists(string $tableName, array $columns): bool {
     global $wpdb;
 
-    $conditions = array_map(function($key) {
-      return $key . '=%s';
+    $conditions = array_map(function($key) use ($columns) {
+      return "$key='{$columns[$key]}'";
```

Comment:
Can we escape here?

Reply:
This is the cumbersome issue we chatted about, escaping here would result in a wrong query